pkg/k8s/client.go

```diff
@@ -31,6 +31,10 @@ func NewKubeClient(kubeconfig, kubecontext string) (*Client, error) {
 		return nil, err
 	}
 
+	if err = kubeClient.validateClusterConnectivity(); err != nil {
+		return nil, fmt.Errorf("couldn't connect to cluster: %s. maybe do you need to connect via VPN?", kubecontext)
```

Comment:
we should use kubeClient.kubecontext, loadConfig will set it to currentContext if no override provided
and set the text to "couldn't connect to context"

Reply:
Fixed